infoschema/tables.go

```diff
@@ -102,6 +102,8 @@ const (
 	TableClusterConfig  = "CLUSTER_CONFIG"
 	tableClusterLoad    = "CLUSTER_LOAD"
 	tableTiFlashReplica = "TIFLASH_REPLICA"
+tableTiDBClusterHardware   = "CLUSTER_HARDWARE"
```

Comment:
Please format these lines by adding tabs before lines.

Reply:
```suggestion
    tableClusterHardware   = "CLUSTER_HARDWARE"
```
Change the name by removing 'TiDB'.